src/main/java/com/alkemy/ong/service/UserServiceImpl.java

```diff
@@ -14,15 +20,21 @@
 import org.springframework.stereotype.Service;
 
 @Service
-public class UserServiceImpl implements UserDetailsService, IDeleteUserService, IGetUserService {
+public class UserServiceImpl implements UserDetailsService, IDeleteUserService, IGetUserService,
+    RegisterUserService {
 
   private static final String USER_NOT_FOUND_MESSAGE = "User not found.";
+  private static final String USER_EMAIL_ERROR = "This email address is already used.";
```

Comment:
TRIVIAL: removería el this ya que no se esta indicando en el mensaje cual es el valor de la dirección.